releaseNotes/KubernetesProvider.md

```diff
@@ -5,7 +5,7 @@
 ## 2.0.0 -
 
 ### Breaking changes
-* Added `auth.workloadIdentity.serviceAccountName` property for workload identity. By default, the Kubernetes Provider uses custom per namespace service account to access Azure App Configuration and Key Vaults, which enhances security isolation. If you want to use global service account, `workloadIdentity.globalServiceAccountEnabled` must be set to `true` at installation time. (see [Use service account](https://learn.microsoft.com/en-us/azure/azure-app-configuration/reference-kubernetes-provider?tabs=default#use-workload-identity) for workload identity authentication)
+* Starting with version 2.0.0, a user-provided service account is required for authenticating with Azure App Configuration using workload identity. This change enhances security through namespace isolation. Previously, a Kubernetes provider’s service account was used for all namespaces. For updated instructions, see the documentation on [using workload identity](https://learn.microsoft.com/en-us/azure/azure-app-configuration/reference-kubernetes-provider?tabs=default#use-workload-identity). If you need time to migrate when upgrading to version 2.0.0, you can temporarily set `workloadIdentity.globalServiceAccountEnabled=true` during provider installation. Please note that support for using the provider’s service account will be deprecated in a future release.

```

Comment:
> using the provider’s service account will be deprecated

The apostrophe in the provider's looks odd, could you double check it's an ASCII character?

Reply:
There's another occurrence in the end :)